### DISCUSSION TITLE
When users will be able to add the library to their projects?

### DISCUSSION THREAD

**Discussion by ArystanK** [mipt-npm/maps-kt#9] [General]
When this library will be puplished and will be available to use?

**Comment by altavir** [comment]
We can publish dev version anytime (one of the snapshot is actually published: https://maven.pkg.jetbrains.space/mipt-npm/p/sci/maven/center/sciprog/). The first version is actually published . But API obviously is not stabilized. I think that we will finalize the current version as soon as somebody says that they want a stable version. 

I will try to fix some issues in the deployment configuration and publish a dev build this week.

**Comment by Lewik** [comment]
You can use is maven local to use it right now
